scripts/digicert-usb-config/eToken-macos.cfg

```diff
@@ -0,0 +1,3 @@
+name=eToken
```

Comment:
I think we should create a `src/windows` directory to put the windows stuff there. This `scripts` directory is not that meaningful. Thoughts?

Maybe something like `src/windows/{build or sign}`

Reply:
I'm OK with that. And all signing js files including jsign.mjs should be moved there as well.